### PR TITLE
llvm 7.0.1 (new formula)

### PR DESCRIPTION
**Pull request by shiro615**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/llvm@7.rb [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    mkdir "build" do
+      system "cmake", "-G", "Unix Makefiles", "..", *(std_cmake_args + args)
+      system "make"
+      system "make", "install"
+      system "make", "install-xcode-toolchain"
+    end
+
+    (share/"clang/tools").install Dir["tools/clang/tools/scan-{build,view}"]
+    (share/"cmake").install "cmake/modules"
+    inreplace "#{share}/clang/tools/scan-build/bin/scan-build", "$RealBin/bin/clang", "#{bin}/clang"
```
Comment (kou) [inline Formula/llvm@7.rb]: It seems that we need to apply #37028 for versioned LLVM formula.

Comment (shiro615) [reply Formula/llvm@7.rb]: Thanks for the information. I've fixed it.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @shiro615

**Comment by csordasmarton** [comment]
@fxcoudert Unfortunately you closed this pull request but not merged. Would you be so kind to merge this?

**Comment by retokromer** [comment]
It was actually merged in https://github.com/Homebrew/homebrew-core/commit/390c83759439aa568e15f6570f0587935b4dd85c.

**Pull request closed without merging**